spec/models/sms/adapters/frontline_sms_adapter_spec.rb

```diff
@@ -35,11 +35,11 @@
   it 'should correctly parse a frontline-style request' do
     Time.zone = ActiveSupport::TimeZone['Saskatchewan']
 
-    request = {'frontline' => '1', 'text' => 'foo', 'from' => '+2348036801489'}
+    request = {'frontline' => '1', 'text' => 'foo', 'from' => '+2348036801489', 'to' => '+123456789'}
```

Comment:
I don't think the to will be included in the request. It should be taken from the mission SMS to number if given. If not, it should be allowed to be null. Both scenarios should probably be tested.